### ISSUE TITLE
p7_Textcnn_train.py中的ValueError

### ISSUE THREAD

**Issue by drinkboy** [brightmart/text_classification#116]
![image](https://user-images.githubusercontent.com/34861107/56565413-e1fd5100-65e2-11e9-9ed9-740b55928fe1.png)

**Comment by jiangjiane** [comment]
predict.extend(logits)
